apis/opts/cpu_test.go

```diff
@@ -5,6 +5,26 @@ import (
 )
 
 func TestValidateCPUPeriod(t *testing.T) {
+
+	tests := []struct {
+		name    string
+		period  int64
+		wantErr bool
+	}{
+		// TODO: Add test cases.
+		{name: "test1", period: 0, wantErr: false},
+		{name: "test2", period: 10, wantErr: true},
+		{name: "test3", period: 2000000, wantErr: true},
+		{name: "test4", period: 1000, wantErr: false},
```

Comment:
How about adding some boundary value here in the test case, like 1000 and 100000? @youmengna